android-kit-base/src/test/kotlin/com/mparticle/kits/KitManagerImplTest.kt

```diff
@@ -843,8 +845,9 @@ class KitManagerImplTest {
             Pair("customerId", "55555"),
             Pair("country", "US")
         )
+
         manager.execute("Test", attributes, null, null, null)
-        Assert.assertEquals(5, attributes.size)
+        Assert.assertEquals(6, attributes.size)
```

Comment:
This was updated to 6, but you don't add the sandbox attribute to look for here.